task08-multiple-languages-optional/methods.js

```diff
@@ -0,0 +1,71 @@
+
+/*Order the list by alphabetical order*/ 
+function sortJSON(data, key) {
+  return data.sort(function (firstElement, secondElement) {
+      var firstCompany = firstElement[key]
+      var secondCompany = secondElement[key];

```

Comment:
This values aren't changing so it is better to use `const` otherwise it's better to use `let` and not `var`